pkg/client/logging/initcontext.go

```diff
@@ -63,3 +66,61 @@ func InitContext(ctx context.Context, name string) (context.Context, error) {
 	}
 	return ctx, nil
 }
+
+func SummarizeLog(ctx context.Context, name string) (string, error) {
+	dir, err := filelocation.AppUserLogDir(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	filename := filepath.Join(dir, name+".log")
+	file, err := os.Open(filename)
+	if err != nil {
+		return "", err
+	}
+	scanner := bufio.NewScanner(file)
+
+	errors := []string{}
+	tail := []string{}
+	for scanner.Scan() {
+		text := scanner.Text()
+		// XXX: is there a better way to detect error lines?
+		parts := strings.Fields(text)
+		if len(parts) > 2 && parts[2] == "error" {
+			errors = append(errors, text)
+			if len(errors) > 10 {
+				errors = errors[len(errors)-10:]
+			}
+		}
+		tail = append(tail, text)
+		if len(tail) > 10 {
```

Comment:
Not sure how useful a `tail -n10` will be.  It's mostly messages about cleaning up and shutting down go-routines.